packages/curve-ui-kit/src/features/connect-wallet/lib/wagmi/useWagmiConfig.ts

```diff
@@ -1,22 +1,29 @@
 import { useMemo } from 'react'
 import type { Chain } from 'viem'
+import { generatePrivateKey } from 'viem/accounts'
 import { mapRecord, recordValues } from '@curvefi/prices-api/objects.util'
 import type { NetworkMapping } from '@ui/utils'
+import { isCypress } from '@ui-kit/utils'
 import { createChainFromNetwork } from './chains'
 import { defaultGetRpcUrls } from './rpc'
 import { createTransportFromNetwork } from './transports'
 import { createWagmiConfig } from './wagmi-config'
+import { createTestConnector } from './wagmi-test'
 
 export const useWagmiConfig = <T extends NetworkMapping>(networks: T | undefined) =>
-  useMemo(
-    () =>
-      networks &&
-      createWagmiConfig({
-        chains: recordValues(networks).map((network) => createChainFromNetwork(network, defaultGetRpcUrls)) as [
-          Chain,
-          ...Chain[],
-        ],
-        transports: mapRecord(networks, (_, network) => createTransportFromNetwork(network, defaultGetRpcUrls)),
+  useMemo(() => {
+    if (networks == null) return
+
+    const chains = recordValues(networks).map((network) => createChainFromNetwork(network, defaultGetRpcUrls)) as [
```

Comment:
Sadly not a nice one-liner anymore, but I need the Ethereum chain created with our custom RPCs in the connector for the Cypress case

Reply:
We created an [RPC enum](https://github.com/curvefi/curve-frontend/blob/a3e27371711ce53b368e15d8d0041661dd242202/packages/curve-ui-kit/src/features/connect-wallet/lib/wagmi/rpc.ts#L6) why not reuse it? It was created exactly to avoid using the ones in curve-js!